api/edge_api/identities/forwarder.py

```diff
@@ -0,0 +1,74 @@
+import json
+
+import requests
+from core.constants import FLAGSMITH_SIGNATURE_HEADER
+from core.signing import sign_payload
+from django.conf import settings
+from rest_framework.request import Request
+
+from environments.dynamodb import DynamoIdentityWrapper
+from util.util import postpone
+
+
+def _should_forward(project_id: int) -> bool:
+    dynamo_identity_wrapper = DynamoIdentityWrapper()
+    return dynamo_identity_wrapper.is_migration_done(project_id)
+
+
+@postpone
+def forward_identity_request(request, project_id: int):
+    forward_identity_request_sync(request, project_id)
+
+
+def forward_identity_request_sync(request, project_id: int):
+    if not _should_forward(project_id):
+        return
+
+    url = settings.EDGE_API_URL + "identities/"
+    if request.method == "POST":
```

Comment:
Perhaps a bit of preference coming in here but I feel like the one of the following would be more pythonic? 

```python
forwarding_method = _forward_identity_post_request if request.method == "POST" else _forward_identity_get_request
forwarding_method(request, url)
```

```python
forwarding_method = {"GET": _forward_identity_get_request, "POST": _forward_identity_post_request}[request.method]
forwarding_method(request, url)
```

```python
forwarding_method = locals()[f"_forward_identity_{request.method.lower()}_request"]
forwarding_method(request, url)
```

Perhaps for the last 2 options, the `_should_forward` method should also check the request.method to ensure it is valid?